README.md

```diff
@@ -21,10 +21,13 @@ IntelliJ has some neat functionality to make running the application very easy.
 #### Set the properties
 The API has a number of properties that need to be set before the applicaiton can run.
 There's a template in `resources/config/application.properties.sample`.
-Copy the contents from this file into a `application-dev.properties`.
+Copy the contents from this file into a `application.properties`.
 This file is gitignored because it contains private information such as possible database credentials and API keys.
 The sample proprties assume a working PostgreSQL installation running in the background.
 
+#### Mailcatcher
+Before you can start the server you should start mailcatcher. This should work by just launching `mailcatcher` from your terminal.
```

Comment:
I have an entire new readme on my machine, I just haven't had the time to include it. It is included in that one. I could open a PR with the new updated readme.